plugin/src/main/scala/org/flywaydb/play/ConfigReader.scala

```diff
@@ -32,49 +35,65 @@ class ConfigReader(configuration: Configuration, environment: Environment) {
   def getFlywayConfigurations: Map[String, FlywayConfiguration] = {
     (for {
       dbName <- getAllDatabaseNames
+      // Will only iterate if this is a configuration group, instead of a single value, allowing easy skipping of irrelevant entries
+      subConfig <- catching(classOf[PlayException]).opt(configuration.getConfig("db." + dbName)).flatten
+
+      // Skip silently if we've been told to
+      if !subConfig.getBoolean("flywayIgnore").getOrElse(false)
```

Comment:
I admit my understanding of the details of how Flyway operates are a little uncertain.

If with no migration files, Flyway would never attempt to create a database connection or even load the driver, I would agree, it's not necessary.  Otherwise, this is a compatibility setting to enable us to inter-operate with other libraries also using this space, and perhaps using drivers not yet supported by flyway.